avm/res/dashboard/grafana/main.bicep

```diff
@@ -0,0 +1,184 @@
+metadata name = 'Azure Managed Grafana'
+metadata description = 'This module deploys an Azure Managed Grafana(Grafana).'
+metadata owner = 'Azure/module-maintainers'
+
+// imports
+import * as grafanaTypes from 'types/grafana.type.bicep'
+
+@description('Required. Name of your Azure Managed Grafana.')
+@minLength(2)
+@maxLength(23)
+param name string
+
+@description('Specifies the resource tags for the Azure Monitor managed service for Prometheus resource.')
```

Comment:
Misses category identifier. In this case it seems to be `Required.`. The same is the case for many other parameters and the CI static tests should give you a list 💪